apps/prairielearn/src/pages/instructorCourseAdminSharing/instructorCourseAdminSharing.html.ts

```diff
@@ -165,10 +165,9 @@ export function InstructorCourseAdminSharing({
                       <button
                         type="button"
                         class="btn btn-xs btn-secondary mx-2"
-                        id="chooseSharingName"
-                        title="Choose Sharing Name"
-                        data-bs-toggle="modal"
-                        data-bs-target="#chooseSharingNameModal"
+                        aria-label="Choose Sharing Name"
```

Comment:
A label is needed because the text content here is hidden on narrow viewports. The same goes for some buttons on the staff page.